### PR TITLE
WP 871 native app platformAgent

### PR DESCRIPTION
**Pull request by mmcgahan**
Native app web views should report a different `platformAgent` in activity recrods than what is usually used for requests from the browser for each web application - `'NATIVE_APP_WEB_VIEW'`

Followup will be to update mup-web and pro-web with this release

### CODE REVIEW

File: packages/mwp-tracking-plugin/src/activity.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ export const getLogger: string => (Object, Object) => mixed = (
 			ip: requestHeaders['remote-addr'] || '',
 			agent: requestHeaders['user-agent'] || '',
 			platform: 'WEB',
-			platformAgent: agent,
+			platformAgent: request.state.isNativeApp ? 'NATIVE_APP_WEB_VIEW' : agent,
```
[the remark this answers is not in the document]
Comment (mmcgahan) [reply packages/mwp-tracking-plugin/src/activity.js]: `request.state` is Hapi's way of referring to cookies in the request - `isNativeApp` is a cookie that is always sent with native app web requests AFAIK

File: packages/mwp-tracking-plugin/src/util/avro.js [hunk 2]
```diff
@@ -137,9 +137,9 @@ const logger = (serializer: Serializer, deserializer: Deserializer) => (
 	const serializedRecord = serializer(record);
 	const deserializedRecord = deserializer(serializedRecord);
 	analyticsLog(serializedRecord);
-	if (process.argv.includes('--debug')) {
-		debugLog(deserializedRecord);
-	}
+	// if (process.argv.includes('--debug')) {
```
[the remark this answers is not in the document]
Comment (mmcgahan) [reply packages/mwp-tracking-plugin/src/util/avro.js]: reinstated, thanks!

File: packages/mwp-tracking-plugin/src/activity.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (burrata) [reply packages/mwp-tracking-plugin/src/activity.js]: Let me double-check, but I think the cookie has to be set manually on the app client side, and we're not using the cookie on mup-web to check right now. Both iOS and the Android are definitely sending query params, though: `isNativeApp=true` and `from_platform=android|ios`.

[the remark this answers is not in the document]
Comment (mmcgahan) [reply packages/mwp-tracking-plugin/src/activity.js]: @burrata if you can confirm this, that would be great - it sounds like the original spec for this ticket isn't quite aligned with chapstick, so will need some review. In particular, it sounds like we need to set `platform`, not `platformAgent`

**Pull request merged**